brax/v2/test_data/colour_objects.xml

```diff
@@ -0,0 +1,33 @@
+<mujoco model="ant">
+  <compiler angle="degree" coordinate="local" inertiafromgeom="true" />
+  <option timestep="0.01" collision="predefined" />
+  <custom>
+    <numeric data="0.0 0.0 0.55 1.0 0.0 0.0 0.0 0.0 1.0 0.0 -1.0 0.0 -1.0 0.0 1.0" name="init_qpos" />
+  </custom>
+  <asset>
+    <mesh name="pyramid" file="meshes/pyramid.stl" scale="0.1 0.1 0.1" />
+  </asset>
+  <default>
+    <joint armature="1" damping="1" limited="true"/>
+    <geom conaffinity="0" condim="3" density="5.0" friction="1 0.5 0.5" margin="0.01" rgba="0.8 0.6 0.4 1"/>
+  </default>
+  <worldbody>
+    <geom conaffinity="1" condim="3" name="floor" pos="0 0 0" rgba="1 0 0.8 1" size="40 40 40" type="plane"/>
+    <body name="sphere" pos="0 0 0.25">
+      <joint axis="1 0 0" name="free1" pos="0 0 0" type="free"/>
+      <geom name="sphere" fromto="-0.5 0.0 0 0.5 0.0 0" pos="0 0 0" size="0.25" type="capsule"/>
```

Comment:
for a sphere `<geom name="sphere" pos="0 0 1" size="0.25" type="sphere"/>`

Reply:
Thank you, good observation, for my test didn´t really mind so I didn´t pay enough attention to that